config/riff-streaming.yaml

```diff
@@ -6567,7 +6567,7 @@ subjects:
 ---
 apiVersion: v1
 data:
-  gatewayImage: bsideup/liiklus:0.9.0
+  gatewayImage: ctr.run/github.com/bsideup/liiklus:00385b688bc34d6fdbe2b5eeb39daa26e09f8b02
```

Comment:
Or wait for https://github.com/bsideup/liiklus/pull/252 to be merged and use a released version

Reply:
you can now use `bsideup/liiklus:0.10.0-rc1`

Reply:
Let's kill two birds with one stone once the streaming-processor PR is merged as well...